db/db.go

```diff
@@ -0,0 +1,7 @@
+package db
+
+import (
+	"github.com/dgraph-io/badger/v3"
+)
+
+var DB *badger.DB
```

Comment:
if you feel paranoid about this, we could always do something like "dependency injection" from the registry point of view.

Reply:
I think this is _ok_ for now. We'll revisit it later.